modules/cmdb/services/audit/audit.go

```diff
@@ -64,7 +62,7 @@ func WithDBClient(db *dao.DBClient) Option {
 }
 
 // WithUCClient 配置 uc client
```

Comment:
Bad annotation.

Reply:
that's old stuff, leave it here, we have a plan to refactor all the code. until then, all chinese text will be deleted.